README.md

```diff
@@ -49,6 +49,25 @@ Given the locking on the **model** there will be some parallelization, but also
 
 Because **Zync** will keep a **log** of **events** it will be able to replay changes and recover the last state just taking last revisions of each **model** and even remove the ones that have been created before but have been deleted.
 
+## Running locally
+
+### Run a PostgreSQL server
+
+It is easy with Docker or Podman
+```
+sudo dnf install postgresql pspg
```

Comment:
The `pspg` part is optional, a matter of taste. Not something you must have for the project. It is a cosmetic nice addon pager. (I do not use it...) Still it does not belong to the project.

By the way postgresql client is not needed, but `libpq-devel`
```suggestion
sudo dnf install postgresql libpq-devel
```

I leave postgresql as it is a must-have client.

Reply:
Thanks, added. I guess I had it already because I didn't notice it missing.